src/paperless/settings.py

```diff
@@ -101,6 +103,9 @@ def __get_boolean(key, default="NO"):
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+# Enable whitenoise compression and caching
+STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
```

Comment:
Is that something I can use without django's runserver, i.e. when I serve static files via Nginx?

Reply:
My understanding is yes, because whitenoise also requires you to collect your static files once and then compresses them but leaves the “original” static files intact.
However, I haven’t tried it yet.
Would it be possible for you to test this on your system? 
Unfortunately I have not gotten Nginx & static files to work properly (-> why I opted for this approach 😅)